contracts/clients/package.json

```diff
@@ -23,17 +23,17 @@
   "dependencies": {
     "@thehubbleproject/bls": "^0.5.1",
     "ethers": "5.5.4",
-    "node-fetch": "2"
+    "node-fetch": "2.6.7"
```

Comment:
I really wish JSON allowed comments... https://stackoverflow.com/questions/69081410/error-err-require-esm-require-of-es-module-not-supported